### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by debasishchakraborty-egovt**

### CODE REVIEW

File: frontend/mgramseva/lib/screeens/AddExpense/AddExpenseWalkThrough/expenseWalkThrough.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ import 'package:mgramseva/widgets/DatePickerFieldBuilder.dart';
 import 'package:mgramseva/widgets/FilePicker.dart';
 import 'package:mgramseva/widgets/SelectFieldBuilder.dart';
 import 'package:mgramseva/widgets/TextFieldBuilder.dart';
-import 'package:mgramseva/widgets/auto_complete.dart';
+import 'package:mgramseva/widgets/AutoCompleteView.dart';
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/screeens/AddExpense/AddExpenseWalkThrough/expenseWalkThrough.dart:8]: +1

**Review by naveen-egov — CHANGES_REQUESTED** [review]
most of the file name are not in snack case  (lowercase letters and underscores between words.) kindly make those  changes

File: frontend/mgramseva/lib/repository/forgot_password_repo.dart [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import 'package:mgramseva/model/forgotPassword/forgot_password.dart';
 import 'package:mgramseva/services/RequestInfo.dart';
-import 'package:mgramseva/services/base_service.dart';
+import 'package:mgramseva/services/BaseService.dart';
```
[the remark this answers is not in the document]
Comment (rahuldevgarg-egov) [reply frontend/mgramseva/lib/repository/forgot_password_repo.dart:3]: https://github.com/egovernments/punjab-mgramseva/pull/426

File: frontend/mgramseva/lib/components/notifications/notifications_list_1.dart
Comment (naveen-egov) [inline frontend/mgramseva/lib/components/notifications/notifications_list_1.dart:1]: Why the file name is 1.dart

File: frontend/mgramseva/lib/screeens/consumer_details/consumer_details_walk_through/walkthrough_1.dart
Comment (naveen-egov) [inline frontend/mgramseva/lib/screeens/consumer_details/consumer_details_walk_through/walkthrough_1.dart:1]: Why is it sufix 1 appended to the file name.

**Review by naveen-egov — APPROVED** [review]

File: frontend/mgramseva/lib/components/notifications/notifications_list_1.dart
Comment (rahuldevgarg) [reply frontend/mgramseva/lib/components/notifications/notifications_list_1.dart:1]: this is duplicate file with minor change

File: frontend/mgramseva/lib/screeens/consumer_details/consumer_details_walk_through/walkthrough_1.dart
Comment (rahuldevgarg) [reply frontend/mgramseva/lib/screeens/consumer_details/consumer_details_walk_through/walkthrough_1.dart:1]: this is duplicate file with minor change

**Review by egov-arindam — APPROVED** [review]

**Pull request merged**